esmvalcore/preprocessor/_volume.py

```diff
@@ -167,28 +167,25 @@ def calculate_volume(cube):
     return grid_volume
 
 
-def average_volume(
+def volume_statistics(
         cube,
-        coord1,
-        coord2,
+        operator,
         fx_files=None):
     """
-    Calculate the average volume.
+    Apply a statistical operation over a volume.
 
     The volume average is weighted acoording to the cell volume. Cell volume
     is calculated from iris's cartography tool multiplied by the cell
     thickness.
 
     Parameters
     ----------
-    cube: iris.cube.Cube
-        input cube.
-    coord1: str
-        name of first coordinate
-    coord2: str
-        name of second coordinate
-    fx_files: dict
-        dictionary of field:filename for the fx_files
+        cube: iris.cube.Cube
+            Input cube.
+        operator: str
+            The operation to apply to the cube.
```

Comment:
Mention what values can be used for this argument

Reply:
I'm planning to add additional functionality here in #1137. For now, this preprocessor can only calculate the mean.

Reply:
```suggestion
            The operation to apply to the cube, options are: 'mean'.
```